catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswSource.java

```diff
@@ -304,33 +312,163 @@ private static JAXBContext initJaxbContext() {
+                value -> cswSourceConfiguration.setUsername((String) value));
+
+        consumerMap.put(CONNECTION_TIMEOUT_PROPERTY,
+                value -> cswSourceConfiguration.setConnectionTimeout((Integer) value));
+
+        consumerMap.put(RECEIVE_TIMEOUT_PROPERTY, value -> cswSourceConfiguration.setReceiveTimeout(
+                (Integer) value));
+
+        consumerMap.put(OUTPUT_SCHEMA_PROPERTY,
+                value -> setConsumerOutputSchemaProperty((String) value));
+
+        consumerMap.put(QUERY_TYPE_QNAME_PROPERTY,
+                value -> cswSourceConfiguration.setQueryTypeQName((String) value));
+
+        consumerMap.put(QUERY_TYPE_PREFIX_PROPERTY,
+                value -> cswSourceConfiguration.setQueryTypePrefix((String) value));
+
+        consumerMap.put(IDENTIFIER_MAPPING_PROPERTY,
+                value -> cswSourceConfiguration.setIdentifierMapping((String) value));
+
+        consumerMap.put(DISABLE_CN_CHECK_PROPERTY,
+                value -> cswSourceConfiguration.setDisableCnCheck((Boolean) value));
+
+        consumerMap.put(COORDINATE_ORDER_PROPERTY, value -> setCoordinateOrder((String) value));
+
+        consumerMap.put(USE_POS_LIST_PROPERTY,
+                value -> cswSourceConfiguration.setUsePosList((Boolean) value));
+
+        consumerMap.put(CREATED_DATE_MAPPING_PROPERTY,
+                value -> cswSourceConfiguration.setCreatedDateMapping((String) value));
+
+        consumerMap.put(EFFECTIVE_DATE_MAPPING_PROPERTY,
+                value -> cswSourceConfiguration.setEffectiveDateMapping((String) value));
+
+        consumerMap.put(MODIFIED_DATE_MAPPING_PROPERTY,
+                value -> cswSourceConfiguration.setModifiedDateMapping((String) value));
+
+        consumerMap.put(CONTENT_TYPE_MAPPING_PROPERTY,
+                value -> setConsumerContentTypeMapping((String) value));
+
+        consumerMap.put(POLL_INTERVAL_PROPERTY, value -> setConsumerPollInterval((Integer) value));
+
+        consumerMap.put(CSWURL_PROPERTY, value -> setConsumerUrlProp((String) value));
+
+        consumerMap.put(IS_CQL_FORCED_PROPERTY,
+                value -> cswSourceConfiguration.setIsCqlForced((Boolean) value));
+
+    }
+
+    /**
+     * Consumer function that sets the cswSourceConfiguration OutputSchema if it is changed.
+     */
+    private void setConsumerOutputSchemaProperty(String schemaProp) {
+        String oldOutputSchema = cswSourceConfiguration.getOutputSchema();
+        cswSourceConfiguration.setOutputSchema(schemaProp);
+
+        LOGGER.debug("{}: new output schema: {}",
+                cswSourceConfiguration.getId(),
+                cswSourceConfiguration.getOutputSchema());
+        LOGGER.debug("{}: old output schema: {}", cswSourceConfiguration.getId(), oldOutputSchema);
+    }
+
+    /**
+     * Consumer function that sets the cswSourceConfiguration ContentType if it is changed.
      */
+    private void setConsumerContentTypeMapping(String currentContentTypeMapping) {
```

Comment:
It's a bit confusing that this parameter is named `currentContentTypeMapping` because it hasn't been set yet. I like the idea of having a `new` prefix for the parameter like you've used in some of the other setters. At the least, it should be consistent.